newIDE/app/src/Export/ExportDialog/ExportLauncher.js

```diff
@@ -133,12 +134,33 @@ export default class ExportLauncher extends Component<Props, State> {
     });
   };
 
+  updateAuthors = async () => {
+    const profile = this.props.authenticatedUser.profile;
+    if (profile) {
+      const authorAcls = this.props.project
+        .getAuthorIds()
+        .toJSArray()
+        .map(authorId => ({
+          userId: authorId,
+          feature: 'author',
+          level: 'owner',
+        }));
+      await setGameUserAcls(
```

Comment:
Nit: try catch and log the error without rethrowing it, so that it does not show as an error screen in development. Rename the function to tryUpdateAuthor to show it's a best effort call.